configs/test/gae/prod3/cron.yaml

```diff
@@ -55,6 +55,11 @@ cron:
     schedule: every day 03:00
     target: cron-service
 
+  - description: Batch FuzzerJobs.
+    url: /batch-fuzzer-jobs
```

Comment:
i thought this list was sorted by url values, so maybe move it after /backup (might also move fuzzer-coverage while you are here).

Reply:
Done!